src/test/resources/__files/body-heads-master-4KaYG.json

```diff
@@ -0,0 +1 @@
+{"ref":"refs/heads/master","url":"https://api.github.com/repos/testOwner/testRepo/git/refs/heads/master","object":{"sha":"815cd56c820c53c665d78f409749a62ad5ec9a48","type":"commit","url":"https://api.github.com/repos/testOwner/testRepo/git/commits/815cd56c820c53c665d78f409749a62ad5ec9a48"}}
```

Comment:
Unfortunately it looks like the server won't serve formatted json files.  It looks like they'll have to stay on one line.